testutil/ciao-down/workload.go

```diff
@@ -75,7 +79,47 @@ func (wkld *workload) save(ws *workspace) error {
 	return nil
 }
 
-func loadWorkloadData(ws *workspace, workloadName string) ([]byte, error) {
+func loadWorkloadData(ctx context.Context, ws *workspace, workloadName string) ([]byte, error) {
+	var absPath string
+	u, err := url.Parse(workloadName)
+	if err == nil {
```

Comment:
Why don't we just return here if we have an error?

url.Parse("ciao") does not return an error so what is the point of falling through to the workload directory scanning code.